<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstrumentManager</class>
 <widget class="QDialog" name="InstrumentManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设备管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_unused">
        <attribute name="title">
         <string>仓库</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,6,2,1,0">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>查找到:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lab_result_1">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>条数据</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="le_instrument_name">
             <property name="placeholderText">
              <string>输入设备名查找</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_search_1">
             <property name="text">
              <string> 查找</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_add">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 170, 0);</string>
             </property>
             <property name="text">
              <string>  新增设备</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="tableView_1"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_department">
        <attribute name="title">
         <string>部门领用情况</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,7,1,2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>查找到:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lab_result_2">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>条数据</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>       部门:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_department_name">
             <item>
              <property name="text">
               <string>全部</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="tableView_2"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_individual">
        <attribute name="title">
         <string>个人领用情况</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0,6,2,1">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>查找到:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lab_result_3">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>条数据</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="le_user_name">
             <property name="placeholderText">
              <string>输入领用人姓名</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_search_3">
             <property name="text">
              <string> 查找</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="tableView_3"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_used">
        <attribute name="title">
         <string>报废设备</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>申报部门:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cb_department">
               <item>
                <property name="text">
                 <string>全部</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>经办人:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cb_jinbanren">
               <item>
                <property name="text">
                 <string>全部</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>审核人:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cb_shenheren">
               <item>
                <property name="text">
                 <string>全部</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,6,1">
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLineEdit" name="le_instrument_name_2">
                 <property name="placeholderText">
                  <string>设备名称</string>
                 </property>
                 <property name="clearButtonEnabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pb_search_4">
                 <property name="text">
                  <string> 查找</string>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="tableView_4"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
